vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -2,445 +2,54 @@
-### Example
-```
----
-kind: "EntandoAppPluginLink"
-apiVersion: "entando.org/v1"
-metadata:
-  name: "test-link"
-  namespace: "your-namespace"
-spec:
-  entandoAppName: your-app
-  entandoAppNamespace: your-namespace
-  entandoPluginName: your-app
-  entandoPluginNamespace: your-namespace
-
-```
-
-### Explanation of properties
-* `spec.entandoAppName` specifies the name of the EntandoApp that requires the plugin, found in `EntandoApp.metadata.name`
-* `spec.entandoAppNamespace` specifies the namespace of the EntandoApp that requires the plugin, found in `EntandoApp.metadata.namespace`
-* `spec.entandoPluginName` specifies the name of the EntandoApp that requires the plugin, found in `EntandoPlugin.metadata.name`
-* `spec.entandoPluginNamespace` specifies the namespace of the EntandoApp that requires the plugin, found in `EntandoPlugin.metadata.namespace`
-
-## EntandoDatabaseService
-
-By default, the Entando Operator deploys one of either the MySQL or PostgreSQL database Docker images for every custom
-resource that requires a database. Many customers may, however, have existing infrastructure for their databases which
-they may want to leverage. The EntandoDatabaseService custom resource allows customers to deploy a Service that points
-to an external database. When deploying one of the Entando Custom Resources that require a database, the Entando
-Operator will look for EntandoDatabases in the same namespace. If it finds one with the same `spec.dbms` setting 
-as the database required by the custom resource, it will create a dedicated schema/username/password combination
-for the custom resource and point the deployment emanating from Custom Resources to this external database.   
-
-
-### Example
-```
----
-kind: "EntandoDatabaseService"
-apiVersion: "entando.org/v1"
-metadata:
-  name: "test-database-service"
-  namespace: "your-namespace"
-spec:
-  dbms: "oracle"
-  host: 10.0.12.41
-  port: 1521
-  databaseName: yourdb 
-  tablespace: 
-  secretName: some-secret
-  jdbcParameters:
-    maxStatements: 300 
-    loginTimeout: 180
-```
+## Entando Core Custom Resource
+The core resource is required for the basic installation of Entando in a Kubernetes cluster. The Entando Operator observes this resource in one or more namespaces. If the core custom resource is created, updated or deleted, the operator will trigger a new run-to-completion pod that implements the state change into the actual Kubernetes cluster. Typically, this results in the deployment of Docker images, services, and sometimes an ingress. Docker images that implement these run-to-completion pods are referred to as Entando Kubernetes Controllers.
+ 
+## General ResourceRequirements Specifications 
+ 
+All the custom resources that result in physical Kubernetes Deployments can be configured with specific resource requirements. These settings can be provided under the `spec` object of the custom resource. It supports the following attributes:
```

Comment:
Remove "physical". I'm not sure how this language made it in originally since there's nothing physical about Kubernetes deployments. Maybe someone meant "an instance"...

Reply:
that's true, didn't even question that part, knowing it's just all electrons floating here and there